### ISSUE TITLE
PropertUnits global merging default set

### ISSUE THREAD

**Issue by imcnally** [brigade/scss-lint#422]
Hey @sds,

Just noticed that PropertyUnits is merging default and local settings for `global`. I tested it by printing the config properties. When I set my local config to:

```yml
  PropertyUnits:
    enabled: true
    global: ['hey']
    properties:
      line-height: []
```

I get: 

```
{
  "enabled"=>true,
  "global"=>["ch", "em", "ex", "rem", "cm", "in", "mm", "pc", "pt", "px", "q", "vh", "vw", "vmin", "vmax", "deg", "grad", "rad", "turn", "ms", "s", "Hz", "kHz", "dpi", "dpcm", "dppx", "%", "hey"],
  "properties"=>{"line-height"=>[]}
}
```

You'll notice my fake unit `'hey'` on the end of the global list.

Merging seems baked into config.rb. It may work for other circumstances, but it works against a specific allowed list in this case.

**Comment by sds** [comment]
Hey @imcnally, thanks for the report.

Our tests didn't catch this because we define configuration directly rather than inheriting from the default configuration. However, it is definitely surprising no one has reported this issue yet, given this has been around since `scss-lint` first added a configuration system.

I've implemented a fix in d59353cb66fa04feca87e47381b4fda7e549062e.

**Comment by bartveneman** [comment]
Had the same issue yesterday, thanks for fixing!

**Comment by imcnally** [comment]
:+1:
